### PR TITLE
Rubrik Polaris Pack

### PR DESCRIPTION
**Pull request by drew-russell**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A new Pack for Rubrik Radar and Sonar. The integration will detect new events in the Rubrik SaaS product and create new incidents based off of those. It also includes two main commands that can be used in Playbooks to pull the information.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by mchasepan** [comment]
Awesome @drew-russell ! Thx for submitting!!

**Comment by drew-russell** [comment]
Ack on the issues from CircleCI. Also working on finishing the partner docs etc.

**Review by abaumgarten — COMMENTED** [review]
@drew-russell, thank you for your contribution!
Please see my comments below, we will discuss them on our next call, I sent you a Slack message to schedule one.
Besides that, here are some general comments: 
1. Please run `demisto-sdk format` on all of your files as all your files are missing some important fields. 
2. Please make some changes for all of your incidentsFilelds files:
*  Change each file name to include your pack name. For example, `incidentfield-RubrikPolaris_Activity_Series_ID.json`. 
* change each incident filed file `id`, `name` and `cliName` to include your pack's name. After this change, verify that it successfully uploaded to Demisto as an incident field.
3. Modify layout files to match the new incident field names/ids.
4. Playbook id and name should be identical.
Feel free to send me a dm on Slack if something is not clear.
note: you should solve all build errors (i.e validate, lint, etc) after changing your code according to my comments.
See you soon.

File: Packs/RubrikPolaris/Integrations/RubrikPolaris/incidentContext.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "progress": "100%",
+    "cluster": {
+        "id": "9f5b9f6c-57a1-4e17-b48c-a610ba1bf979",
+        "name": "shrd1-rbk01"
+    },
+    "message": {
+        "text": "Detected anomalous filesystem activity with High confidence and High levels of encryption (File Change: 1655 Added, 36 Modified, 1650 Removed)",
+        "id": "36a4c91c-0774-457b-879d-8f0e31dd8bfe"
+    }
+}
```
[the remark this answers is not in the document]
Comment (drew-russell) [reply Packs/RubrikPolaris/Integrations/RubrikPolaris/incidentContext.json:21]: Honestly, not sure. It was automatically downloaded by the demisto-sdk. Will remove for now and re-add if there are any issues.

File: Packs/RubrikPolaris/Scripts/RubrikPolarisSonarSensitivteHits/RubrikSonarSensitiveHits.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Only needed for local development
+import demistomock as demisto
+from CommonServerPython import *
+import traceback
+
+
+def main() -> None:
+    try:
+
+        sonar_context = demisto.context()["Rubrik"]["Sonar"]
```
[the remark this answers is not in the document]
Comment (drew-russell) [reply Packs/RubrikPolaris/Scripts/RubrikPolarisSonarSensitivteHits/RubrikSonarSensitiveHits.py:10]: Add a try/except to be on the safe side.

File: Packs/RubrikPolaris/Scripts/RubrikPolarisSonarSensitivteHits/RubrikSonarSensitiveHits.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            },
+            {
+                'stat': 'Files with Hits',
+                'result': sonar_context["filesWithHits"]
+            },
+            {
+                'stat': 'OpenAccess Stale Files',
+                'result': sonar_context["openAccessStaleFiles"]
+            },
+        ]
```
[the remark this answers is not in the document]
Comment (drew-russell) [reply Packs/RubrikPolaris/Scripts/RubrikPolarisSonarSensitivteHits/RubrikSonarSensitiveHits.py:46]: Covered by the try/except I added with the above comment

File: Packs/RubrikPolaris/Scripts/RubrikSonarSensitiveHits/RubrikSonarSensitiveHits.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def main() -> None:
+    try:
+        try:
```
Comment (abaumgarten) [inline Packs/RubrikPolaris/Scripts/RubrikSonarSensitiveHits/RubrikSonarSensitiveHits.py:7]: You don't need to use `try` twice, only once.

**Review by abaumgarten — COMMENTED** [review]
Please see that

**Review by abaumgarten — APPROVED** [review]

**Comment by abaumgarten** [comment]
@drew-russell Thank you for your contribution!

**Pull request merged**